h/templates/deform/mapping_item.jinja2

```diff
@@ -8,7 +8,9 @@
             {% if field.error %}is-error{% endif %}
             {% if show_char_counter %} js-character-limit {%- endif %}
             {% if field.schema.hide_until_form_active %} is-hidden-when-loading {%- endif %}"
-     {%- if field.description -%}
+     {%- if field.title -%}
+     title="{{ _(field.title) }}"
```

Comment:
Hmm... I think the [`title` attribute of an HTML element](https://html.spec.whatwg.org/multipage/dom.html#the-title-attribute) and [`title` field of a `colander.SchemaNode`](https://docs.pylonsproject.org/projects/colander/en/latest/api.html?highlight=SchemaNode#colander.SchemaNode) have slightly different purposes. The former is "advisory information for the element, such as would be appropriate for a tooltip." (which I take as up to say, a sentence in length) whereas `field.title` is for the visible title of a field which tends to be shorter.

I think what would be preferable here is to strip HTML tags from the description when rendering the `title` attribute, using a [Jinja filter](http://jinja.pocoo.org/docs/2.10/templates/#filters)